recipe/patch_yaml/pandas.yaml

```diff
@@ -12,8 +12,17 @@ if:
   version_le: 1.14.2
 then:
   - replace_depends:
-      old: python-dateutil >=2.7.3
       new: python-dateutil >=2.8.1
+      old: python-dateutil >=2.7.3
   - replace_depends:
-      old: pytz >=2017.2
       new: pytz >=2020.1
+      old: pytz >=2017.2
```

Comment:
merging from main gives the expected patch, but I'm confused as to why this change here is needed.

Reply:
These changes were caused by a formatting tool but I thought they didn't matter. I can remove them if you think it's better to keep the old fotmat